ansible_collections/arista/avd/molecule/eos_cli_config_gen_deprecated_vars/inventory/host_vars/host1/ethernet-interfaces-switchport.yml

```diff
@@ -0,0 +1,38 @@
+---
+### Ethernet Interfaces ###
+
+# Testing deprecated keys for switchport config in ethernet_interfaces
```

Comment:
Please list the keys that are deprecated and that they will be removed in 6.0.0. Thanks

Reply:
same in port-channels

Reply:
Fixed